experimental/dds/tree2/src/core/tree/anchorSet.ts

```diff
@@ -402,59 +400,48 @@ export class AnchorSet implements ISubscribable<AnchorSetRootEvents> {
 		}
 	}
 
+	private nextRange = 0;
+	private createEmptyDetachedField(): FieldKey {
+		const detached: DetachedField = brand(String(this.nextRange++));
+		const key = detachedFieldAsKey(detached);
+		assert(!this.root.children.has(key), 0x370 /* new range must not already exist */);
```

Comment:
the assert tag 0x370 is already in use by the code you copied this from. This will cause this code to fail the asserts policy check. You should do:

```suggestion
		assert(!this.root.children.has(key), "new range must not already exist");
```
and let who ever does the next round of assert tagging allocate a new code for it.